README.md

```diff
@@ -0,0 +1,10 @@
+###Build
```

Comment:
Not for this PR, but can you update the README to feel like the other READMEs and also add a LICENSE?